forem/forem
# Remove :runtime_banner feature flag

## What type of PR is this? (check all applicable)


## Description

This PR removes the `runtime_banner` FeatureFlag, making the banner visible to all iOS users (Android or Desktop browsers shouldn't be affected by this).

The PR also makes a small adjustment to the positioning of the Banner: Making it slightly bigger and positioned lower on the screen.

## Related Tickets & Documents

https://github.com/forem/rfcs/pull/72

## QA Instructions, Screenshots, Recordings

- Boot the app locally and browse it using an iOS device
   - The banner should be visible only to iOS devices and not other platforms
- You can also remove the `Browser-iOS` class from the banner itself (in `app/views/layouts/application.html.erb`)
   - Then you can use the DevTools to browse as if using a mobile device from your desktop browser

#### Before (positioning)

<img width="666" alt="Screen Shot 2021-05-05 at 06 21 09" src="https://user-images.githubusercontent.com/6045239/117140027-22633e00-ad6a-11eb-85ce-f9f503f43a94.png">

#### After (positioning)

<img width="846" alt="Screen Shot 2021-05-27 at 17 12 37" src="https://user-images.githubusercontent.com/6045239/119907968-bf9b3780-bf0e-11eb-9579-069bc0e66d40.png">

### UI accessibility concerns?

None because it's not modifying the existing elements

## Added tests?

- [x] No, and this is why: Tests already exist and they are only modified to work without the FeatureFlag

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: This is part of the Forem iOS app release and will be communicated with the help of the Marketing team

## [optional] Are there any post deployment tasks we need to perform?

None

## [optional] What gif best describes this PR or how it makes you feel?

![human billboard](https://media.giphy.com/media/l0HlzhAVF2bewwBig/giphy.gif)
